### ISSUE TITLE
Document the convention for generating an Amendment ID

### ISSUE THREAD

**Issue by intelliot** [XRPLF/xrpl-dev-portal#1761]
This would be a good place for it: https://xrpl.org/amendments.html

By convention, it is the sha512half hash of the amendment name. https://github.com/xpring-eng/xrp-ledger-countdown/blob/e773351bfb0be282bfd950652a373f294f27965d/index.js#L50

**Comment by kennyzlei** [comment]
@intelliot your code link seems to point to a private github org, can you relink something that is public?

**Comment by intelliot** [comment]
Good catch. I searched for, and was not able to find, a public alternative. However, I reviewed the code linked, and I think it's OK to open source it. So I went ahead and made it public.

**Comment by ckniffen** [comment]
Also found here. https://github.com/ripple/explorer/blob/staging/src/containers/shared/amendmentUtils.ts

**Comment by amarantha-k** [comment]
Related: Issue #2644 to document the non-admin feature RPC command.
